python/lsst/sdm/schemas/efd_latiss.yaml

```diff
@@ -0,0 +1,2764 @@
+    datatype: float
+    nullable: true
+    description: Standard deviation of the standard deviation of wind speed (x, y, z) estimated from quartiles along axis 2 (salIndex 204).
+  - name: "at_salindex204_wind_speed_speedstddev_2_min"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_speedstddev_2_min"
+    datatype: float
+    nullable: true
+    description: Minimum standard deviation of wind speed (x, y, z) estimated from quartiles along axis 2 (salIndex 204).
+  - name: "at_salindex204_wind_speed_speedstddev_2_max"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_speedstddev_2_max"
+    datatype: float
+    nullable: true
+    description: Maximum standard deviation of wind speed (x, y, z) estimated from quartiles along axis 2 (salIndex 204).
+  - name: "at_salindex204_wind_speed_magnitude_mean"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_magnitude_mean"
+    datatype: float
+    nullable: true
+    description: Mean median wind speed magnitude (salIndex 204).
+  - name: "at_salindex204_wind_speed_magnitude_stddev"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_magnitude_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the median wind speed magnitude (salIndex 204).
+  - name: "at_salindex204_wind_speed_magnitude_min"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_magnitude_min"
+    datatype: float
+    nullable: true
+    description: Minimum median wind speed magnitude (salIndex 204)s
+  - name: "at_salindex204_wind_speed_magnitude_max"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_magnitude_max"
+    datatype: float
+    nullable: true
+    description: Maximum median wind speed magnitude (salIndex 204).
+  - name: "at_salindex204_wind_speed_maxmagnitude_mean"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_maxmagnitude_mean"
+    datatype: float
+    nullable: true
+    description: Mean maximum wind speed magnitude (salIndex 204).
+  - name: "at_salindex204_wind_speed_maxmagnitude_stddev"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_maxmagnitude_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the maximum wind speed magnitude (salIndex 204).
+  - name: "at_salindex204_wind_speed_maxmagnitude_min"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_maxmagnitude_min"
+    datatype: float
+    nullable: true
+    description: Minimum maximum wind speed magnitude (salIndex 204).
+  - name: "at_salindex204_wind_speed_maxmagnitude_max"
+    "@id": "#exposure_efd.at_salindex204_wind_speed_maxmagnitude_max"
+    datatype: float
+    nullable: true
+    description: Maximum maximum wind speed magnitude (salIndex 204).
+  - name: "at_salindex205_wind_speed_0_mean"
+    "@id": "#exposure_efd.at_salindex205_wind_speed_0_mean"
+    datatype: float
+    nullable: true
+    description: Mean median wind speed (x, y, z) in AuxTel-GillLabJack01 along axis 0 (salIndex 205).
```

Comment:
General question: Is "axis 0" here "x?" If so would it be possible to include that in the description instead? (Same for other similar columns.)

Reply:
This needs to be updated by the scientists team.